spec.bs

```diff
@@ -1175,7 +1253,7 @@ To <dfn>convert a modifier to a string</dfn> given a [=part/modifier=] |modifier
   1. If |protocolValue| is null or the empty string, then set |protocolValue| to "`https`".
   1. Set |dummyURL|'s [=url/scheme=] to |protocolValue|.
      <p class="note">Note, we set the [=URL record=]'s [=url/scheme=] in order to check if it [=is special=].  If |protocolValue| is missing we purposely default the URL to being special by using "`https`" for the [=url/scheme=].</p>
-  1. If |dummyURL| [=is special=], then return the result of running [=canonicalize a standard pathname=] given |pathnameValue|.
+  1. If |protocolValue| is a [=special scheme=], then return the result of running [=canonicalize a standard pathname=] given |pathnameValue|.
```

Comment:
Hmm, can we just get rid of all steps related to dummyURL? It looks like you're just creating it but never using it.